.travis.yml

```diff
@@ -17,6 +17,10 @@ cache:
 before_script:
   - NODE_ENV=$BUILD_ENV npm run build
 
+script:
+  # Running front-end tests on the oldest Node version we support
+  - if [ "${TRAVIS_NODE_VERSION}" = "4" ]; then npm test; else npm run travis; fi
```

Comment:
No v4 anymore

Reply:
I've removed this, as other things caused it to bring nothing (we saved 4 seconds, yay...)